app/Http/Controllers/ActorController.php

```diff
@@ -49,7 +49,7 @@ public function store(Request $request)
     {
         $this->authorize('create', Actor::class);
 
-        $validated = $request->validate([
+        $request->validate([
             'name' => 'bail|required|string|max:255',
```

Comment:
don't recognize this syntax, what does it do?

Reply:
I assume you mean on line 53? It's a regular array key value mapping. I define the valid input from the request.